x-pack/plugins/security_solution/public/resolver/view/panels/copyable_panel_field.tsx

```diff
@@ -69,15 +92,13 @@ export const CopyablePanelField = memo(
           isOpen={isOpen}
           panelPaddingSize="s"
         >
-          <EuiToolTip
-            content={i18n.translate('xpack.securitySolution.resolver.panel.copyToClipboard', {
-              defaultMessage: 'Copy to Clipboard',
-            })}
-          >
-            <WithCopyToClipboard
-              data-test-subj="resolver:panel:copy-to-clipboard"
-              text={textToCopy}
-              titleSummary={textToCopy}
+          <EuiToolTip content={TRANSLATED_COPY_TO_CLIPBOARD}>
+            <EuiButtonIcon
+              aria-label={TRANSLATED_COPY_TO_CLIPBOARD}
```

Comment:
❔ I think you could either put the text that's going to be copied in that `aria-label` like `Copy ${textToCopy} to clipboard` or you could maybe add an aria-description="${texttoCopy}" .

Reply:
aria-description doesn't seem to be valid. I can change it to `Copy ${textToCopy} to clipboard`?